templates/chapter_summaries/architecture_summary.html

```diff
@@ -113,59 +97,41 @@ <h1 class="ons-u-mt-m">Code and Architecture Summary</h1>
                         try {
```

Comment:
It looks like we can remove this? I don't see any operations now you have removed the errorPanel stuff